iothub/service/tests/DeviceAuthenticationTests.cs

```diff
@@ -278,13 +278,13 @@ public async Task DeviceAuthenticationGoodAuthSHA256()
                 mockHttpRequestFactory,
                 s_retryHandler);
 
-            await devicesClient.CreateAsync(deviceBadAuthConfig).ConfigureAwait(false);
+            await devicesClient.CreateAsync(deviceGoodAuthConfig).ConfigureAwait(false);
         }
 
         [TestMethod]
-        public async Task DeviceAuthenticationIsCertificateAuthority()
+        public async Task DeviceAuthentication_CertificateAuthConfigTest5()
```

Comment:
Can we come up with a meaningful name about the case this test covers without having to use a numeric suffix?

Reply:
Absolutely!